packages/create-fuels/src/cli.ts

```diff
@@ -158,10 +160,11 @@ export const runScaffoldCli = async ({
   // Note: `pnpm run xprebuild` -> rewritten to -> `pnpm run prebuild` (on prePublish script)
   const packageJsonPath = join(projectPath, 'package.json');
   const packageJsonContents = readFileSync(packageJsonPath, 'utf-8');
-  const newPackageJsonContents = packageJsonContents.replace(
-    `pnpm run prebuild`,
-    packageManager.run('prebuild')
-  );
+  const fuelsVersion = getPackageVersion(args);
```

Comment:
Can we use the version directly from the `versions` package?

```suggestion
  const fuelsVersion = versions.FUELS;
```